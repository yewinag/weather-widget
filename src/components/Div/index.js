import styled from 'styled-components';

const Div = styled.div`
    padding: 10;
    align-items: center;
`
export default Div;