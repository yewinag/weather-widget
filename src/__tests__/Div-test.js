import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Div from '../components/Div';

Enzyme.configure({ adapter: new Adapter() });

describe('div layout', () => {
    it('render div', () => {
        const wrapper = shallow(<Div />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});