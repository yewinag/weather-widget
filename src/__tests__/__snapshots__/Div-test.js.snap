// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`div layout render div 1`] = `ShallowWrapper {}`;
